swan-lake/learn/calling-java-code-from-ballerina.md

```diff
@@ -84,17 +84,18 @@ This section assumes that you have already read [How to Structure Ballerina Code
 
 #### Creating a Ballerina Project
 ```sh
-> ballerina new yaml-project
-Created new Ballerina project at yaml-project
+> ballerina new yaml_project
+Created new Ballerina project at yaml_project
 
 Next:
```

Comment:
We dont have this message now. Please refer to the message from the latest master

Reply:
Added the new message.